File: week3/Student.cpp
#include<iostream>
using namespace std;

class Student
{
    public:
        string fullName;
        int rollNum;
        double semPerentage;
        string collegeName;
        int collegeCode;
        Student()
        {
            cout<<"Student class is created!!\nIam a student\n";
        }
        ~Student(){}
};

int main()
{
    Student renu;
    cout<<"Enter your sweet name : ";
    getline(cin,renu.fullName);
    cout<<"Enter your roll number : ";
    cin>>renu.rollNum;
    cout<<"Enter your sem percentage : ";
    cin>>renu.semPerentage;
    cout<<"Enter your college name : ";
    cin>>renu.collegeName;
    cout<<"Enter your college code : ";
    cin>>renu.collegeCode;
    cout<<"Name : "<<renu.fullName<<endl;
    cout<<"Roll Num : "<<renu.rollNum<<endl;
    cout<<"Sem Percentage : "<<renu.semPerentage<<endl;
    cout<<"College Name : "<<renu.collegeName<<endl;
    cout<<"College Code : "<<renu.collegeCode<<endl;
    return 0;
}

File: Student.java
import java.util.Scanner;

class Student54
{
  public
    String fullName;
    int rollNum;
    double semPercentage;
    String collegeName;
    int collegeCode;
    Student54()
    {
      System.out.println("Student class is created!!\nIam a student\n");
    }
    protected void finalize() throws Throwable
    {
      System.out.println("The class is finalized\n");
    }
}

public class Student
{
  public static void main(String[] args)
  {
    Student54 vanitha=new Student54();;
    Scanner input=new Scanner(System.in);
    System.out.print("Enter your sweet name : ");
    vanitha.fullName=input.nextLine();
    System.out.print("Enter your rollNumber : ");
    vanitha.rollNum=input.nextInt();
    System.out.print("Enter your semPercentage : ");
    vanitha.semPercentage=input.nextDouble();
    input.nextLine();
    System.out.print("Enter your collegeName : ");
    vanitha.collegeName=input.nextLine();
    System.out.print("Enter your collegeCode : ");
    vanitha.collegeCode=input.nextInt();
    System.out.print("Name : "+vanitha.fullName+"\n");
    System.out.print("RollNumber : "+vanitha.rollNum+"\n");
    System.out.print("SemPercentage : "+vanitha.semPercentage+"\n");
    System.out.print("CollegeName : "+vanitha.collegeName+"\n");
    System.out.print("CollegeCode : "+vanitha.collegeCode+"\n");
    input.close();
  }
}
